pkg/application/consul/consul.go

```diff
@@ -0,0 +1,82 @@
+package consul
+
+import (
+	"github.com/awslabs/eksdemo/pkg/application"
+	"github.com/awslabs/eksdemo/pkg/cmd"
+	"github.com/awslabs/eksdemo/pkg/installer"
+	"github.com/awslabs/eksdemo/pkg/template"
+)
+
+// https://developer.hashicorp.com/consul/docs/k8s/installation/install
+// https://developer.hashicorp.com/consul/tutorials/kubernetes/kubernetes-eks-aws
+// https://artifacthub.io/packages/helm/hashicorp/consul
+
+func NewApp() *application.Application {
+	app := &application.Application{
+		Command: cmd.Command{
+			Name:        "consul",
+			Description: "HashiCorp Consul Service-Mesh",
+		},
+
+		Installer: &installer.HelmInstaller{
+			ChartName:     "consul",
+			ReleaseName:   "consul",
+			RepositoryURL: "https://helm.releases.hashicorp.com",
+			VersionField:  "1.4.1",
```

Comment:
`VersionField` is not used. I need to remove this from the `HelmInstaller` struct. Can you remove it?